redisolar/dao/redis/metric.py

```diff
@@ -56,9 +54,9 @@ def _get_measurements_for_date(self, site_id: int, date: datetime.datetime,
         # e.g.: metrics:whU:2020-01-01:1
         key = self.key_schema.day_metric_key(site_id, unit, date)
 
-        # Return a reverse range so that we're always consuming from the end
+        # Use negative indexes to get `count` number of items from the end
         # of the sorted set.
-        metrics = self.redis.zrevrange(key, 0, count - 1, withscores=True)
+        metrics = self.redis.zrange(key, -(count), -1, withscores=True)
 
         for metric in metrics:
             # `zrevrange()` returns (member, score) tuples, and within these
```

Comment:
Comment about `zrevrange` here will need updating to reflect use of `zrange`.

Reply:
Fixed both of these!